starter-code/views/gallery.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Lab Express basic Website - GALLERY</title>
+    <link rel ="stylesheet" href="/css/style.css">
+</head>
+<body>
+
+<header>
+        <a href="/">Home</a>
+        <h1>THE CHUCK NORRIRS GALLERY </h1>
+        <a href="/about">Gallery</a>
```

Comment:
You made a slight mistake here, you linked to the about page but said that it was meant to go to the gallery. Unless... maybe it was a plan....